King-Benx/news-watchtower-5
# Implement swipe to refresh, code refactor(OOP) [Delivers #163857069]

https://www.pivotaltracker.com/story/show/163857069

**Overview**
This PR delivers the user with the ability to refresh the list of news by dragging down the screen.

**Background**
NONE

**Acceptance Criteria**
- The application should operate as before.
- News list should be updated once the screen is dragged down.

**How to manually test**
- Clone the repository
- Checkout to the ft-swipe-to-refresh-163857069
- Run ./gradlew build
- Run the application on an emulator or device
- Drag/pull the screen down and observe.

**Screenshots**
<img width="95" alt="swipetorefresh" src="https://user-images.githubusercontent.com/17830204/52560217-a850e280-2e08-11e9-9cf4-81fcc5acae78.png">

**Story**
[#163857069]
